lib/src/generator/generator_utils.dart

```diff
@@ -13,31 +13,35 @@ import 'package:dartdoc/src/model/model_element.dart';
 /// Convenience function to generate category JSON since different generators
 /// will likely want the same content for this.
 String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
-  var encoder = pretty ? JsonEncoder.withIndent(' ') : JsonEncoder();
   // ignore: omit_local_variable_types
-  final List<Map<String, Object>> indexItems =
-      categories.map((Categorization e) {
-    var data = <String, Object>{
-      'name': e.name,
-      'qualifiedName': e.fullyQualifiedName,
-      'href': e.href,
-      'type': e.kind,
-    };
+  final List<Map<String, Object>> indexItems = categories
+      .map((Categorization cat) {
```

Comment:
optional: I'd prefer to just expand all these into full names, e.g. `categorization` or `indexable` or `enclosedElement`, but if that seems unwieldly to you I don't feel strongly on it.